.github/workflows/backend.yml

```diff
@@ -88,6 +88,12 @@ jobs:
           # https://github.com/codecov/codecov-bash/blob/7100762afbc822b91806a6574658129fe0d23a7d/codecov#L891
           fetch-depth: '2'
 
+      - name: Update environment for silo databases
+        if: |
+          contains( github.event.pull_request.labels.*.name, 'Trigger: Silo db' )
+        run: |
+          echo "SENTRY_USE_SILOS=1" >> "$GITHUB_ENV"
```

Comment:
should we put this in `setup-sentry` directly?

Reply:
I don't know. I'm hoping this is a short-lived label based workflow. Once we have master green with split databases, I'd like to make it the default behavior as it won't add overhead and will prevent new cross db operations from being introduced.